src/coreclr/src/jit/compiler.cpp

```diff
@@ -2962,6 +2962,12 @@ void Compiler::compInitOptions(JitFlags* jitFlags)
             opts.dspDiffable = true;
         }
 
+        // This one applies to both Ngen/Jit Disasm output: COMPlus_JitDasmWithAddress=1
+        if (JitConfig.DasmWithAddress() != 0)
+        {
```

Comment:
Doesn't this need to be `JitDasmWithAddress()` now?

Reply:
Ah yes, I forgot to update it.